RecoHI/HiJetAlgos/plugins/HiSignalGenJetProducer.cc

```diff
@@ -0,0 +1,128 @@
+// -*- C++ -*-
+//
+// Package:    HiSignalGenJetProducer
+// Class:      HiSignalGenJetProducer
+//
+/**\class HiSignalGenJetProducer HiSignalGenJetProducer.cc yetkin/HiSignalGenJetProducer/src/HiSignalGenJetProducer.cc
+
+ Description: <one line class summary>
+
+ Implementation:
+     <Notes on implementation>
+*/
+//
+// Original Author:  Yetkin Yilmaz
+//         Created:  Tue Jul 21 04:26:01 EDT 2009
+//
+//
+
+// system include files
+#include <memory>
+#include <vector>
+
+// user include files
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "FWCore/Framework/interface/EDProducer.h"
+
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/Utilities/interface/InputTag.h"
+
+#include "DataFormats/Common/interface/View.h"
+#include "DataFormats/JetReco/interface/GenJetCollection.h"
+#include "DataFormats/GeometryVector/interface/VectorUtil.h"
+#include "DataFormats/HepMCCandidate/interface/GenParticle.h"
+
+#include "DataFormats/Math/interface/Point3D.h"
```

Comment:
These two includes do not seem to be used here: please remove if so
Perhaps, also some of the includes above are not needed: please check